PICMI_Python/particles.py

```diff
@@ -402,6 +407,7 @@ def __init__(self, flux, flux_normal_axis,
                  directed_velocity = [0.,0.,0.],
                  flux_tmin = None,
                  flux_tmax = None,
+                 gaussian_flux_momentum_distribution = True,
```

Comment:
As a general rule, it is better not to have default values in the picmi standard, but to rely on the default values in the implementing codes.